pubsub/integration_test.go

```diff
@@ -1263,11 +1264,17 @@ func TestIntegration_OrderedKeys_JSON(t *testing.T) {
 		key := parts[0]
 		msg := parts[1]
 		publishData = append(publishData, testutil2.OrderedKeyMsg{Key: key, Data: msg})
-		topic.Publish(ctx, &Message{
-			ID:          msg,
+		r := topic.Publish(ctx, &Message{
 			Data:        []byte(msg),
 			OrderingKey: key,
 		})
+		go func() {
+			_, err := r.Get(ctx)
+			if err != nil {
+				// Can't fail inside goroutine, so just log the error
+				log.Printf("publish error for message(%s): %v", msg, err)
```

Comment:
Might be worth just using the [test logger](https://pkg.go.dev/testing#T.Logf):
```suggestion
				t.Logf("publish error for message(%s): %v", msg, err)
```